curriculum/challenges/english/blocks/es-a1-quiz-spanish-fundamentals/68d2a8fa725f6db0b7215411.md

```diff
@@ -222,20 +222,20 @@ What letter corresponds to the name `efe`?
 
 #### --text--
 
-How many letters are in the Spanish alphabet?
+Choose the option that matches the numbers to their correct names.
 
 #### --distractors--
 
-17 letters
+`1 - Uno` `5 - Cinco` `6 - Ocho` `0 - Nueve`
```

Comment:
Could we use a comma to separate the numbers?

We ran into issues with Crowdin before where it didn't like seeing two adjacent backticked segments (#60039). I'm not sure if that is still a problem, but to be safe:

```suggestion
`1 - Uno`, `5 - Cinco`, `6 - Ocho`, `0 - Nueve`
```